spring-batch-infrastructure/src/test/java/org/springframework/batch/item/database/support/AbstractSqlPagingQueryProviderTests.java

```diff
@@ -104,6 +104,51 @@ public void testGenerateJumpToItemQueryForFirstPageWithMultipleSortKeys() {
 		assertEquals(getJumpToItemQueryForFirstPageWithMultipleSortKeys(), s);
 	}
 	
+	@Test
+	public void testRemoveKeyWordsFollowedBySpaceChar() {
+		String selectWithSpace = "SELECT id, 'yes', false";
+		String fromWithSpace = "FROM test.verification_table";
+		String whereWithSpace = "WHERE TRUE";
+		pagingQueryProvider.setSelectClause( selectWithSpace );
+		pagingQueryProvider.setFromClause( fromWithSpace );
+		pagingQueryProvider.setWhereClause( whereWithSpace );
+		
+		
+		assertEquals("id, 'yes', false", pagingQueryProvider.getSelectClause());
+		assertEquals("test.verification_table", pagingQueryProvider.getFromClause());
+		assertEquals("TRUE", pagingQueryProvider.getWhereClause());
+	}
+
+	@Test
+	public void testRemoveKeyWordsFollowedByTabChar() {
+		String selectWithSpace = "SELECT\tid, 'yes', false";
```

Comment:
This variable should be named `selectWithTab`, do you agree?

```suggestion
		String selectWithTab = "SELECT\tid, 'yes', false";
```

Same for `fromWithSpace` and `whereWithSpace`. But I think it is better to use something like `selectClause`, `fromClause` and `whereClause` since the test name is already explicit enough about the case we are testing. Wdyt?

Reply:
You are certainly correct. I will get this fixed.